ui/apps/dashboard/src/components/Metrics/utils.ts

```diff
@@ -93,9 +99,16 @@ export const getLineChartOptions = (
       // Attach tooltips to a dedicated dom node above interim parents
       // with low z-indexes
       appendTo: () => document.getElementById('chart-tooltip'),
-      extraCssText: 'max-height: 350px; overflow-y: scroll;',
-      className: 'no-scrollbar',
-      transitionDuration: 1.5,
+      transitionDuration: 1,
+      //
+      // rendering our tooltip componen to a string doesn't support click events
+      // so wrap the whole thing in a vanilla js clipboard cliip handler
+      formatter: (params: any) =>
+        `<div onclick="navigator.clipboard.writeText('${
```

Comment:
A bit of shenanigans here. I'm rendering the thing to a string so we can use an actual component with all the tailwind styles and icon trimmings. But we can send events along that way so I'm just wrapping it up in a vanilla js event handler.